src/sprite.c

```diff
@@ -1481,6 +1481,21 @@ u16 LoadSpriteSheetByTemplate(const struct SpriteTemplate *template, u32 frame,
     sheet.tag = template->tileTag;
     return LoadSpriteSheetWithOffset(&sheet, offset);
 }
+// Like LoadSpriteSheet, but checks if already, and uses template image frames
```

Comment:
Add newline before comment to separate from previous function

Reply:
Done locally